ethexe/signer/Cargo.toml

```diff
@@ -24,4 +24,4 @@ secp256k1 = { version = "0.30", features = ["rand", "global-context", "hashes",
 sha3 = { version = "0.10", default-features = false }
 
 [dev-dependencies]
-alloy.workspace = true
+alloy = { workspace = true, features = ["k256"] }
```

Comment:
why implementing it within the pr instead of opening another one?

Reply:
That was a part of signed related refactoring and historically was considered to be something easy and fast to merge. Now we are here.